features/git-sync-all/no_conflicts/feature_branches/without_open_changes.feature

```diff
@@ -0,0 +1,39 @@
+Feature: git-sync-all: syncs all feature branches with open changes
```

Comment:
You flipped the "with/without open changes" description. This spec is without open changes, but internally says it is with. Same for the other file.